Routes/doctor.js

```diff
@@ -0,0 +1,437 @@
+ *                 noOfRooms:
+ *                     type: number
+ *                 roomCharge:
+ *                     type: number
+ *          bill:
+ *             type: object
+ *             properties:
+ *                 prescriptionId:
+ *                      type: string
+ *                 doctorCharge:
+ *                      type: number
+ *                 noOfDays:
+ *                      type: number
+ *                 operationCharge:
+ *                      type: number                     
+ *          
+ *             
+ */
+
+/**
+ * @swagger
+ * /doctor/newDoctor:
+ *  post:
+ *      summary: Registering a new Doctor  
+ *      description: Add the new doctor data in database
+ *      requestBody:
+ *          required: true
+ *          content:
+ *              application/json:
+ *                  schema:
+ *                      $ref: '#/components/schemas/Doctor'
+ *      responses:
+ *          200:
+ *              description: Added Successfully
+ *              
+ */ 
+
+DoctorRoute.post('/newDoctor',async (req,res) => {
+    const hospital = await Hospital.findById(req.body.hospitalId)
+    if(hospital != null || hospital != undefined){
+
+        const doctor = new Doctors({
+            doctorName: req.body.doctorName,
+            doctorEmail: req.body.doctorEmail,
+            doctorPassword: await bcrypt.hash(req.body.doctorPassword,10),
+            doctorDepartment: req.body.doctorDepartment,
+            hospitalId: req.body.hospitalId,
+            hospitalName: hospital.hospitalName
+            
+        })
+        const newDoctor = await doctor.save()
+        res.json(newDoctor);  
+    }
+    else{
+        res.json({result: "incorrect hospital id try again"})
+    }
+    
+})
+
+/**
+ * @swagger
+ * /doctor/hospital:
+ *  post:
+ *      summary: Registering a new hospital 
+ *      description: Add the new hospital data in database
+ *      requestBody:
+ *          required: true
+ *          content:
+ *              application/json:
+ *                  schema:
+ *                      $ref: '#/components/schemas/Hospital'
+ *      responses:
+ *          200:
+ *              description: Added Successfully
+ *              
+ */ 
+
+DoctorRoute.post('/hospital',verifyToken, async (req,res) =>{
+    const hospital = new Hospital({
+        hospitalName: req.body.hospitalName,
+        hospitalAddress: req.body.hospitalAddress,
+        noOfRooms: req.body.noOfRooms,
+        roomCharge: req.body.roomCharge,
+        
+    })
+    const newhospital = await hospital.save()
+    res.json(newhospital); 
+})
+
+/**
+ * @swagger
+ * /doctor/newMedicine:
+ *  post:
+ *      summary: Registering a new Medicine 
+ *      description: Add the new medicine data in database
+ *      requestBody:
+ *          required: true
+ *          content:
+ *              application/json:
+ *                  schema:
+ *                      $ref: '#/components/schemas/Medicine'
+ *      responses:
```

Comment:
add `- tag` to all the API docs, so that they are properly structured

Reply:
added tags [tags](https://github.com/Sankalp-Sachapara/HospitalPractice/pull/2/files/893ce87fe4fe8a5be2c037597a3a6cfbdffc56ac#r1045074312)